.travis.yml

```diff
@@ -25,11 +25,14 @@ before_install:
   - curl https://github.com/open-io/ansible-role-openio-gridinit/blob/master/library/gridinitcmd.py --create-dirs -o library/gridinitcmd.py
   - sudo pip install --upgrade pip
   - sudo pip install ansible-lint yamllint
+  - export IPVAGRANT=$(ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1)
```

Comment:
"VAGRANT" WTF ?

Reply:
TravisCi uses vagrant

Reply:
OK, thought it was docker-based, showing my ignorance...